### ISSUE TITLE
If can't find lat lon, it returns an empty subset file.  It should fail instead

### ISSUE THREAD

**Issue by jamesfwood** [podaac/l2ss-py#308]
This request should fail:

https://harmony.earthdata.nasa.gov/C2936721448-POCLOUD/ogc-api-coverages/1.0.0/collections/parameter_vars/coverage/rangeset?forceAsync=true&subset=lat(-82.29044999999999%3A82.08044999999998)&subset=lon(4.51755%3A175.50045)&granuleId=G3062447313-POCLOUD&variable=all

But it returns an empty subset file.  No good.

**Comment by frankinspace** [comment]
Returning an empty file was a design choice suggested by podaac user group. Was this discussed further re: https://bugs.earthdata.nasa.gov/browse/TRT-36 ??

**Comment by jamesfwood** [comment]
> Returning an empty file was a design choice suggested by podaac user group. Was this discussed further re: https://bugs.earthdata.nasa.gov/browse/TRT-36 ??

Yeah we should return empty in most situations.  @sliu008 please investigate why it's failing because of the lat and lon.  Also, we pass in the granule that should be within that bbox too, so we need to figure out why this is returning empty.

**Comment by frankinspace** [comment]
Depends on implementation of https://bugs.earthdata.nasa.gov/browse/HARMONY-1989?src=confmacro

**Comment by frankinspace** [comment]
@jamesfwood @sliu008 This should be actionable now; harmony introduced a new exception that should be thrown if there is no data.

**Comment by jamesfwood** [comment]
I think this was failing because of a different reason than "no data".  Now the request returns a valid subset file with data.  I think it may have been a umm-v issue.
